pages/landing.tsx

```diff
@@ -18,7 +18,7 @@ const Landing: FC = () => {
       <h2 className="my-14">{t('description')}</h2>
       <div className="flex justify-center flex-wrap text-xl gap-4">
         <div id="withoutESRF">
-          <LinkButton href="/email" text={t('withoutESRF')}></LinkButton>
+          <LinkButton href="/consent" text={t('withoutESRF')}></LinkButton>
```

Comment:
Translation key should be in kebab case `without-esrf`. We can review the translation in another PR if it's perferred.